src/OpenAI.csproj

```diff
@@ -7,7 +7,7 @@
     <VersionPrefix>2.0.0</VersionPrefix>
     <VersionSuffix>beta.7</VersionSuffix>
 
-    <TargetFrameworks>netstandard2.0;net6.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
```

Comment:
This may cause issues with sync-over-async in the SCM pipeline, which relies on newer net6.0 sync APIs being available for HttpClient APIs to avoid sync-over-async for the majority case.  Removing this target would require sync-over-async for all sync HttpClient calls.

Reply:
I don't think we will start doing sync over async, unless this OpenAI dll calls the sync HTTP APIs directly. The System.ClientModel implementation (6+ or NS2) is picked based on the runtime of the program, not based on framework target of OpenAI. @ericstj, could you confirm or deny?

Reply:
Having said that, I don't think we should be removing NET0 target. I think we should always cross target to the latest LTS target.